N new.csv

```diff
@@ -0,0 +1,22 @@
+Lemma,Sense,Word Class,Year,Quotation,Source,Note
+奶糖,/,noun,1886,,洪士提反譯《萬國藥方》卷一,
+難產,/,noun,1822,LABOUR（Difficult labour，拆副難產；難產. There is a volume on difficult labours, called 達生編 馬禮遜 英華字典,
+腦海,/,noun,1913,brains ,翻譯字典 Mateer Ada Haven Hand Book of New Terms and Newspaper Chinese,
+腦子,/,noun,unknown,腦子像石頭，總是說不通。,魯迅(1881-1936)《書信集‧致孟十還》, publication date absent in Schmidt
+笑劇,/,noun,unknown,那些官兒，也一定是笑劇中的腳色。,魯迅(1881-1936)《書信集‧致姚克》, publication date absent in Schmidt
+內部,/,noun,1878,,郭嵩燾《倫敦與巴黎日記》卷廿七,
+內果皮,/,noun,1858,果皮的最後一層,翻譯資料 植物學,
+內容,/,noun,1839,,林則徐《嚴禁本地人與外人非法往來交易告示》, not clear if the phrase quoted from this source has the same meaning as '事物內部所含的實質或意義。'
+內在,/,noun,1916,immanent,翻譯字典 赫美玲 官話,
+尿道,/,noun,1869,passage(the passage of the urine),翻譯字典 羅存德 英華字典,
+凝固,/,noun,1848,TO CONDENSE (to harden), 麥都思 英華字典,
+濃度,/,noun,1908,Strength（Quality of being strong， 堅强之性質；active power or vigour of an animal body.力，氣力，力； 見今基本語義對 firmness，堅牢，堅固，穩固；solidity，堅實；toughness，堅韌；power or vigour of any kind，各種精力；power ofresisting attacks， 抵持之力：power of stay， 支持之力：power of mind，心力： intellectual force，腦力，智力： robustness， 强壯；spirit， 精神； animation，活潑之氣；force of writing，筆力，文勢，文辭之勢力； nervous diction，義詞詞組， 辭句激感之力；vividness，繪聲繪色，活潑；physical virtue，體力；legal force，法律之效力；natural force，天然之力；amount of force, military or naval，兵力（水兵或陸兵）：vehemence，猛烈，奮發，踴躍；intensity， 濃度；thatquality which tends to secure results， 可致效果之力：one who, or that which, is regarded as embodying oraffording force, strength or firmness， 有力或有堅性之人或物，泰山，磐石；as, the strength of law， 法律之勢力；God is our refuge and strength，上帝為我之保障為我之磐石； the strength of iron， 鐵之堅力：the strengthof spirits，酒之濃度，酒力；the strength of the castle， 礮臺之抵持力；the strength of electricity， 電力；what isthe strength of the enemy？敵人之兵力如何：great strength，大力。重力，鼎力、骨力：of little strength， 少、力怯；strength of mind，心力，壯志；bodily strength， 身力：the strength of a bow， 弓力；the strength of wine，酒之濃淡，酒力；with united strength，協力，同力，併力：prostrate strength， 力薄；to exert one's strength，竭、出力、勉力、番力，用力、窮力；strength exhausted，力窮，力竭，力盡；the strength of the bones， 骨力：thestrength ofa fort，做盛之堅固：to give strength，增力，補力，加力：Iknow his Strength，我知其力；strength ofreason， 道理之力，道理之眞實；upon the strength of，憑：strength of materials， 材料强弱）,顏惠慶 英華大辭典,
```

Comment:
Your job isn’t just to copy whatever Schmidt gives you. You need to exercise your editorial judgement when Schmidt produces gibberish like this.

Ask yourself: if you’re publishing a dictionary, would you paste all this straight in?

Include only the relevant material. Always imagine you’re working under space constraints.

Reply:
Absolutely, let me make the change now. Thanks!